content/en/security/cloud_security_management/misconfigurations/custom_rules.md

```diff
@@ -34,17 +34,17 @@ To view the list of the available resource types for your custom rules, see [Clo
 
 To clone a rule:
 
-1. Find the rule you want to copy one of the following ways:
-   - Navigate to [**Security** > **Configuration** > **Cloud Security Management** > **Compliance Rules**][1]. Select a rule you want to copy to open its details page.
-   - Navigate to [**Security** > **Cloud Security Management** > **Explorer** > **Misconfigurations**][2]. Select a misconfiguration to open its details, and select **Edit Rule** from the **Rule** menu.
+1. Find the rule you want to copy by doing one of the following:
+   - Navigate to the [**Misconfigurations Rules**][1] page. Select a rule you want to copy to open its details page.
+   - Navigate to the [**Misconfigurations Explorer**][2]. Select a misconfiguration to open its details, then select **Edit Rule** from the **Rule** menu.
```

Comment:
Ignore if incorrect. I see the **Edit Rule** as a standalone button
```suggestion
   - Navigate to the [**Misconfigurations Explorer**][2]. Select a misconfiguration to open its details, then select **Edit Rule**.
```

Reply:
Good catch!